core/constants.py

```diff
@@ -3,14 +3,21 @@
 import math
 import json
 
+# Autonomy General Configuration
+EVENT_LOOP_DELAY = 0.1
```

Comment:
seconds? if you are going to give everything else a unit, use one here too. Especially since most time in code is in ms

```suggestion
EVENT_LOOP_DELAY = 0.1  # seconds
```